locales/fa.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "شما؟ (الان انتخاب کنید، بعداً می‌توانید تغییر دهید) ",
+	"select-user-type.driver": "راننده",
+	"select-user-type.passenger": "مسافر",
+	"request-phone.type_your_phone": "لطفاً شماره‌ی خود را ارسال کنید. این شماره بدون اجازه‌ی شما به اشتراک گذاشته نخواهد شد",
+	"request-phone.button_text": "ارسال شماره",
+	"request-phone.all_set": "👌 تمام! الان میتوانید درخواست راننده کنید",
+	"passenger-index.choose_taxi_type": "چه نوع تاکسی‌ای میخواهید؟",
+	"passenger-index.car": "🚗 ماشین",
+	"passenger-index.motorbike": "🏍 اسکوتر/ موتورسیکلت",
+	"passenger-index.s": "🔧 تنظیمات",
+	"passenger-index.error_only_known_type": "انتخاب نامعتبر، لطفاً ماشین یا موتورسیلکت را انتخاب کنید یا به تنظیمات بروید",
+	"passenger-request-location.provide_location": "برای یافتن رانندگان اطراف، لطفاً موقعیت خود را به اشتراک بگذارید",
+	"passenger-request-destination.provide_destination": "مقصد شما؟ لطفاً آدرس دقیق خیابان را وارد کنید (تا جهت یاب جی‌پی‌اس آن را پیدا کند) یا مقصد خود را بنویسید",
+	"order-submitted.order_submitted": "⏳ در جستجوی راننده ها...اگر راننده خود را پیدا کردید، کلید زیر را فشار دهید 👇",
+	"order-submitted.cancel": "پایان جستجو",
+	"order-submitted.error_incorrect_input": "انتخاب نامعتبر، لطفاً دوباره تلاش کیند",
+	"order-cancelled.order_cancelled": "👌 تشکر، پایان جستجوی راننده‌ها",
+	"driver-select-vehicle-type.select": "نوع ماشینی که به عنوان تاکسی از آن استفاده می کنید را انتخاب کنید",
+	"driver-select-vehicle-type.motorbike": "🏍 اسکوتر / موتورسیلکت",
+	"driver-select-vehicle-type.car": "🚗 ماشین",
+	"driver-select-vehicle-type.error_only_known_type": "انتخاب نامعتبر، لطفاً ماشین یا اسکوتر / موتورسیلکت را انتخاب کنید",
+	"driver-explain-checkins.text": "تقریباً تمام!، چند نکته: تلگرام پشتیبانی لحظه‌ای موقعیت جی‌پی‌اس ندارد، بنابراین هر زمان موقعیت خود را تغییر می دهید، دوباره وارد شوید یا اگر آماده دریافت درخواست‌های جدید هستید هر یک ساعت دوباره وارد شوید. این به ما کمک مییکند تا مسافران اطراف شما را پیدا کنیم. روی /"بعدی/" کلیک کنید ",
```

Comment:
@Alir3z4 I've tested a fully working version of the bot with your translations. Nice job :)